contrib/sssd.spec.in

```diff
@@ -224,7 +224,7 @@ Requires: libsss_certmap = %{version}-%{release}
 # required by sss_analyze
 Requires: python3-systemd
 Requires: python3-click
-Recommends: sssd-dbus
+Suggests: sssd-dbus
```

Comment:
Or is it actually "Requires:"?

Reply:
Hi,

yes, I think `Requires` would be better because lots of the sssctl functionality depends on it. In older versions we had an implicit `Requires` via `libsss_simpleifp`.

bye,
Sumit

Reply:
Thanks, changed.